.github/workflows/integration-tests.yml

```diff
@@ -10,15 +10,31 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-20.04
+    name: test ${{ matrix.event_loop_manager }} (${{ matrix.python_version }})
     if: "!contains(github.event.pull_request.labels.*.name, 'disable-integration-tests')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python-version: "3.8"
+            event_loop_manager: "libev"
+
+          - os: ubuntu-latest
+            python-version: "3.8"
+            event_loop_manager: "asyncio"
```

Comment:
This is not going to pass. I think we should still test with asyncore with 3.8, until we can replace it with asyncio.

Reply:
I know it won't, I want to see how bad it is

Reply:
anyhow we should also cover the others to the extent possible

Reply:
> I know it won't, I want to see how bad it is

It will most likely be very bad, asyncio backend doesn't work at all yet

Reply:
and now I've run into https://github.com/eventlet/eventlet/issues/812
we can't run tests  with python 3.12...

Reply:
Even tests with different connection class than eventlet?